js/components/utils/DataUtils.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @description Some utility functions that can be shared across UI components
+ */
+
+/* eslint-disable import/prefer-default-export */
+
+export const checkBooleanString: any = (inputString) => typeof inputString === "boolean" ? inputString :
```

Comment:
Should the inputString argument not be typecast as a (string | boolean)?

Reply:
Yeah, I thought about doing that, but went instead for just returning false if the function were given anything but a string or a boolean.
I think it's probably best to add types though as you suggest, seems bad just to return false for invalid types.